src/utils/generic-utils.ts

```diff
@@ -77,6 +77,32 @@ namespace GenericUtils {
 
     return parseFloat(number ?? "0.0").toFixed(precision);
   };
+
+  /**
+   * Converts CSS HEX color to CSS RGB function
+   *
+   * If the input is already in RGB format, it will be returned as is.
+   * If the input is not a valid HEX, undefined will be returned.
+   *
+   * @param hexColor hex color string
+   * @returns rgb color string
+   */
+  export const hexToRGB = (hexColor: string) => {
```

Comment:
Maybe you should use some kind of color library to abstract parsing e.g. https://www.npmjs.com/package/color-parse

Reply:
Resolved